src/components/Application/ApplicationTitle.tsx

```diff
@@ -2,29 +2,52 @@ import React from "react";
 import { HeaderName as CarbonHeaderName } from "carbon-components-react/lib/components/UIShell";
 import { CLASSPREFIX as eccgui } from "../../configuration/constants";
 
+type depSvg = HTMLElement & SVGElement;
```

Comment:
We should decide on naming conventions for type aliases. Imho they should start with an upper case like interfaces and classes.